### ISSUE TITLE
spaced-comment: how to enable for inline comments only?

### ISSUE THREAD

**Issue by puzrin** [eslint/eslint#5071]
eslint 2.0.0-beta.2

Is it possible to enable `spaced-comment` rule for inline comments only?

```js
//that should be a error
/*that should not be a error*/
```

**Comment by eslintbot** [comment]
@puzrin Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by nzakas** [comment]
I don't believe this is possible right now.

**Comment by nzakas** [comment]
Unfortunately, it looks like consensus couldn't be reached on this issue and so I'm closing it. While we wish we'd be able to accommodate everyone's requests, we do need to prioritize. We've found that issues failing to reach consensus after 21 days tend never to reach consensus, and as such, we close those issues. This doesn't mean the idea isn't interesting, just that it's not something the team can commit to.

**Comment by puzrin** [comment]
Hm... Is it possible to give special label to useful issues, closed with such reason?
